HabboHotel/Groups/GroupManager.cs

```diff
@@ -169,4 +171,34 @@ public List<Group> GetGroupsForUser(int userId)
         }
         return groups;
     }
+
+    public Dictionary<int, string>? GetHabboGroupBadges(Habbo habbo)

```

Comment:
Method name doesnt reflect what its actually doing. This doesnt get the GroupBadges for a given Habbo.
This gets the badges for all users in a room.
The Habbo isn't needed as argument. Pass the room instead.